src/main/webapp/app/core/websocket/websocket.service.ts

```diff
@@ -166,6 +167,7 @@ export class JhiWebsocketService implements OnDestroy {
             if (channel != null && (!Object.keys(this.myListeners).length || !this.myListeners.hasOwnProperty(channel))) {
                 this.myListeners[channel] = this.createListener(channel);
             }
+            // TODO: Fix access of myListeners[undefined]
```

Comment:
Can someone help me with this fix access of `myListeners[undefined]`?

Reply:
i removed the optional. this caused only one problem in the app. i fixed that issue